### ISSUE TITLE
Fatal error when using cluster_leading_eigen function

### ISSUE THREAD

**Issue by yangli8377** [igraph/rigraph#203]
I was using cluster_leading_eigen function for community detection. I keep getting a fatal error. For example, if I try the examples in the help file,

g <- make_full_graph(5) %du% make_full_graph(5) %du% make_full_graph(5)
g <- add_edges(g, c(1,6, 1,11, 6, 11))
lec <- cluster_leading_eigen(g)

The third line will give a fatal error and restart the R session. What is wrong here?

**Comment by fairmiracle** [comment]
yes, I got the same problem. 'cluster_leading_eigen' seems not working on the new R 3.4.0 but OK with R 3.2.5 and 3.3.1

**Comment by gaborcsardi** [comment]
Try the `igraph_1.1.0.zip` file from here: https://github.com/igraph/rigraph/releases/tag/v1.1.0

**Comment by yangli8377** [comment]
I installed the 1.1.0 version. When I library igraph, I got

> library("igraph", lib.loc="~/R/win-library/3.4")
Error: package or namespace load failed for ‘igraph’ in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]):
 there is no package called ‘pkgconfig’

**Comment by gaborcsardi** [comment]
It depends on the `pkgconfig` package, you need to install that, too.

**Comment by yangli8377** [comment]
Thanks. It's working now.

**Comment by MichaelChirico** [comment]
I just encountered this on version 1.2.11 (so, more recent than this issue).

It appears fixed on CRAN (1.3.5 as of now) but I don't see any associated NEWS item, it seems pretty important given the R session crashes.

Here's an MRE in case the bug was fixed incidentally and it can serve as a regression test:

```r
library(igraph)
g = make_graph("Zachary")
cluster_leading_eigen(g, callback = function(...) stop())
```

cc @ntamas in case you don't get notifications for the old / closed issue.
